pkg/gatherers/clusterconfig/pod_network_connectivity_checks.go

```diff
@@ -11,17 +11,31 @@ import (
 	"github.com/openshift/insights-operator/pkg/record"
 )
 
-// GatherPNCC collects a summary of failed PodNetworkConnectivityChecks from
-// last 24 hours.
+// GatherPNCC Collects a summary of failed PodNetworkConnectivityChecks from last 24 hours.
+//
 // Time of the most recently failed check with each reason and message is recorded.
 //
-// Resource API: podnetworkconnectivitychecks.controlplane.operator.openshift.io/v1alpha1
-// Docs for relevant types: https://pkg.go.dev/github.com/openshift/api/operatorcontrolplane/v1alpha1
+// ### API Reference
+// - podnetworkconnectivitychecks.controlplane.operator.openshift.io/v1alpha1
+// - https://pkg.go.dev/github.com/openshift/api/operatorcontrolplane/v1alpha1
+//
+// ### Sample data
+// - docs/insights-archive-sample/config/podnetworkconnectivitychecks.json
+//
+// ### Location in archive
+// - `config/podnetworkconnectivitychecks.json`
+//
+// ### Config ID
+// `clusterconfig/pod_network_connectivity_checks`
+//
+// ### Released version
+// - 4.8.2
```

Comment:
I've seen this a couple of times here and it's probably not a big deal, but I am wondering about the `4.8.0` vs `4.8.2`

Reply:
Yeah, I found a lot of places where the RN points to `4.8.2` and not to `4.8.0`. If you are curious [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1949907)

Reply:
Is it possible that `4.8.2` was the first GA/public release of 4.8?

Reply:
Yep. It was.